pandas/_libs/tslibs/offsets.pyx

```diff
@@ -99,12 +98,6 @@ def apply_index_wraps(func):
     # do @functools.wraps(func) manually since it doesn't work on cdef funcs
     wrapper.__name__ = func.__name__
     wrapper.__doc__ = func.__doc__
-    try:
-        wrapper.__module__ = func.__module__
```

Comment:
Any reason for removing this? I think module is still part of the python data model

https://docs.python.org/3/reference/datamodel.html

Reply:
This `try` never succeeds